packages/server/scripts/notify-unverified-anj-registrations.js

```diff
@@ -15,17 +15,26 @@ dotenv.config()
 
 async function main() {
   const users = await User.findUnverifiedAnjRegistrations()
+  const emails = {}
   for (const user of users) {
     const { email: { email }, address } = user
-    await emailClient.sendEmailWithTemplate({
-      To: email,
-      TemplateAlias: 'notification-settings-announcement',
-      TemplateModel: {
-        dashboardUrl: 'https://court.aragon.org/dashboard',
-      },
-    })
+    if (emails[email]) continue
+    emails[email] = 1
```

Comment:
For what it's worth... this is where a `Set()` might be better :). We may also want to lowercase if we haven't already.

Reply:
Fixed!